src/wrapper/objects/auto_array.rs

```diff
@@ -0,0 +1,193 @@
+use log::debug;
+
+use crate::{errors::*, objects::JObject, sys, JNIEnv};
+use combine::lib::any::TypeId;
```

Comment:
```suggestion
use std::any::TypeId;
```
`conbine` only reexports `TypeId` from `std, right?